ceph_iscsi_config/settings.py

```diff
@@ -23,6 +24,9 @@ def init():
     config = Settings()
 
 
+MON_CONFIG_PREFIX = 'config://'
```

Comment:
Nit: perhaps just to be more explicit about the source:
```suggestion
MON_CONFIG_PREFIX = 'mon-config-key://'
```

Reply:
Yeah, I agree that 'config://' isn't the most explicit or obvious prefix. Only chose it as it was already chosen in RGW to do the exact same thing, so thought keeping things same across the project would make it clearer and easier to operate.

Happy to have my arm twisted though ;)

Reply:
Ah -- didn't realize it was in RGW. If that's the case, objection withdrawn